### ISSUE TITLE
AI should always choose X≥1 when casting cards with X in the mana cost

### ISSUE THREAD

**Issue by Ebola16** [magefree/mage#5894]
Astral Cornucopia is usually useless when X=0. The AI should cast this card with X≥1.

Update: Maybe we should have the AI always choose X≥1 so it doesn't cast useless cards.
Here is a list of all cards with X in the mana cost: https://gatherer.wizards.com/Pages/Search/Default.aspx?action=advanced&mana=+~[X]

**Comment by Zerrisx** [comment]
Duplicate of #6839

**Comment by ddzn** [comment]
It can be useful to get cards into the graveyard, like when paying for delve, increasing storm count or to grow a Tarmogoyf.

**Comment by Ebola16** [comment]
Yes but those strategies are a little too advanced for the current AI. This suggestion would prevent the AI from casting useless cards.
